benchmark/cuchembm/metrics/utils.py

```diff
@@ -0,0 +1,312 @@
+#!/opt/conda/envs/rapids/bin/python3
```

Comment:
There are a number of files that should show up as having been moved but don't. Makes it difficul tot understand what's changed.

Reply:
All files in the PR where moved. Apart from that, most of the files had to be modified to accommodate for the changes to dataset classes.